98 Commented/00 Intro/00 Boilerplate/webpack.config.js

```diff
@@ -0,0 +1,43 @@
+/*
+WEBPACK is a module bundler for modern JavaScript applications. 
+When webpack processes your application, it recursively builds a dependency graph,
+that includes every module your application needs, then packages all of those modules into one or more bundles.
+
+There are a lot of statements that have been standardized in ES2015. 
+Although they are not supported in most browsers yet, webpack does support them out of the box.
+
+Webpack "transpiles" the code so that older browsers can also run it.
+*/
+
+module.exports = {
+    //An entry point indicates which module webpack should use to begin building out its internal dependency graph
+    // So, webpack will start reading on this entry to build our app, web, etc.
```

Comment:
We can add something like, in this sample app "students.js" is the starting point, usually you will find a more generica root  file name like _index.js_, _app.js_....

Reply:
Ok, changes written and commit done with #comment2 (this changes has been made on the three projects that I had downloaded)